src/mpid/ch4/netmod/ofi/ofi_init.c

```diff
@@ -74,9 +74,32 @@ categories :
       verbosity   : MPI_T_VERBOSITY_USER_BASIC
       scope       : MPI_T_SCOPE_LOCAL
       description : >-
+        This variable is only provided for backward compatibility. When using OFI versions 1.5+, use
+        the other memory region variables.
+
         If true, MR_SCALABLE for OFI memory regions.
         If false, MR_BASIC for OFI memory regions.
 
+    - name        : MPIR_CVAR_CH4_OFI_ENABLE_MR_VIRT_ADDRESS
+      category    : CH4_OFI
+      type        : int
+      default     : -1
+      class       : device
+      verbosity   : MPI_T_VERBOSITY_USER_BASIC
+      scope       : MPI_T_SCOPE_LOCAL
+      description : >-
+        If true, enable virtual addressing with MR_BASIC for OFI memory regions.
```

Comment:
Do we need to mention `MR_BASIC` here? Seems more confusing to the unfamiliar. Also, I would add "This variable is used for OFI versions 1.5+."

Reply:
Yep. Good point. I updated the description for this variable and the one below.